image/rhel/Dockerfile

```diff
@@ -91,6 +71,27 @@ RUN ln -s entrypoint-wrapper.sh /stackrox/admission-control && \
     mkdir -p /var/cache/stackrox && chown -R 4000:4000 /var/cache/stackrox && \
     chown -R 4000:4000 /tmp
 
+COPY --from=stackrox_data /stackrox-data /stackrox/static-data
+COPY ./docs/api/v1/swagger.json /stackrox/static-data/docs/api/v1/swagger.json
+
+COPY bin/central            /stackrox/central
+COPY bin/migrator           /stackrox/bin/migrator
+COPY bin/compliance         /stackrox/bin/compliance
+COPY bin/kubernetes-sensor  /stackrox/bin/kubernetes-sensor
+COPY bin/sensor-upgrader    /stackrox/bin/sensor-upgrader
+COPY bin/admission-control  /stackrox/bin/admission-control
+COPY bin/roxctl*            /assets/downloads/cli/
+
+COPY THIRD_PARTY_NOTICES /THIRD_PARTY_NOTICES/
```

Comment:
I'd move `THIRD_PARTY_NOTICES` above, closer to `static-data`, but it's tiny anyway.